### PR TITLE
Show number of hits when the user hovers on a line number

### PR DESCRIPTION
**Pull request by jaril**
image

### CODE REVIEW

File: src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  const setHoveredLineNumber = event => {
+    let target = event.target;
+    const isBreakpointMarkerNode = target.closest(".new-breakpoint");
+
+    // If hovered on a breakpoint marker, get the corresponding linenumber element.
+    if (isBreakpointMarkerNode) {
+      const gutterNode = target.closest(".Codemirror-gutter-elt");
+      target = gutterNode?.previousElementSibling;
+    }
```
Comment (jasonLaster) [inline src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.js:31]: could we move this logic to gutter-events too so we just emit the line number? This is cute, but I don't think we want to think about other gutter markers in this api, just the gutter el and line


```
const setHoveredLineNumber = (line, target) => {
    runAnalysisOnLine(cx, line);
    setLineNumberNode(target);
}
```

File: src/protocol/logpoint.ts [hunk 2]
```diff
@@ -251,11 +263,12 @@ export function setLogpointByURL(
   line: number,
   column: number,
   text: string,
-  condition: string
+  condition: string,
+  showInConsole: boolean = true
```
Comment (jasonLaster) [inline src/protocol/logpoint.ts:267]: i wonder if `setLogpoint` should take an object? these long lists of args gets sketchy

```
logGroupId, mapper, null, true
```

File: src/protocol/logpoint.ts [hunk 3]
```diff
@@ -300,7 +313,7 @@ function eventLogpointMapper(getFrameworkListeners: boolean) {
 
 export async function setEventLogpoint(logGroupId: string, eventTypes: string[]) {
   const mapper = eventLogpointMapper(/* getFrameworkListeners */ true);
-  const analysisId = await createLogpointAnalysis(logGroupId, mapper, null);
+  const analysisId = await createLogpointAnalysis(logGroupId, mapper, null, true);
```
Comment (jasonLaster) [inline src/protocol/logpoint.ts:316]: should condition be null? probably, but we need to update the type..

**Review by jasonLaster — COMMENTED** [review]
couple comments. otherwise great

File: src/protocol/logpoint.ts (hunk 3 above)
Comment (jaril) [reply src/protocol/logpoint.ts:316]: did you mean the logpoint parameter?

**Pull request merged**